process/cg-transition.html

```diff
@@ -106,50 +106,39 @@ <h3>
     <h3>
       Are IPR commitments in place?
     </h3>
-    <p>
-      Community Groups have a "two-step IPR policy" so that people make
-      lightweight commitments when work starts and more comprehensive
-      commitments once work is mature.
-    </p>
+    <p>When a Working Group plans to take up a Community Group specification, it is important that there be "IPR safety" around the specification managed by the Working Group. Relevant questions are:</p>
+
     <ul>
-      <li>Initially, a participant makes certain commitments regarding their
-      own Contribution under the <a href=
-      "https://www.w3.org/community/about/process/cla/">W3C Community
-      Contributor Licensing Agreement (CLA)</a>.
-      </li>
-      <li>Later, a participant can make a voluntary commitment regarding the
-      entire specification under the <a href=
-      "https://www.w3.org/community/about/agreements/final/">W3C Community
-      Final Specification Agreement (FSA)</a>.
-      </li>
+      <li>Who has made significant contributions to the Community Group specification? See below for information about a <a href="#contributor-list">tool to identify significant contributors</a>.</li>
+      <li>Have those contributors (or the organizations they represent) joined the Working Group or do they plan to do so?</li>
     </ul>
-    <p>
-      By comparison, under the W3C Patent Policy for Working Groups,
-      participants make full specification commitments upon joining the group.
-    </p>
-    <p id="ipr-commitments">
-      There are thus three ways to procure commitments by Community Group
-      contributors to the full text of a specification transferred to a Working
-      Group:
+
+    <p id="ipr-commitments">If significant contributors have not
+      joined the Working Group or don't plan to, then steps should be
+      taken to secure IPR commitments from them. There are three ways for CG
+      contributors to make commitments for the specification managed
+      by the Working Group:
```

Comment:
```suggestion
    <p id="ipr-commitments">There are three ways for CG
      contributors to make commitments for the specification managed
      by the Working Group:
```
It seems to me that the "If" clause is redundant with the first bullet

Reply:
it also seems to me that the two bullets above that paragraph ("who as made significant…") are a repeat of what this paragraph is saying in a more accurate way